salsah1/SalsahBuild.sbt

```diff
@@ -75,8 +75,8 @@ lazy val javaHeadlessTestOptions = Seq(
     "-Dconfig.resource=headless-testing.conf"
 ) ++ javaTestOptions
 
-lazy val akkaVersion = "2.5.9"
-lazy val akkaHttpVersion = "10.0.11"
+lazy val akkaVersion = "2.5.11"
+lazy val akkaHttpVersion = "10.1.0"
```

Comment:
Are the Akka stability issues we had fixed? What about the load tests that we wanted to execute?

Reply:
this is a release PR. those issues are handled in other issues.

Reply:
But the release PR is merged into develop, right?

Reply:
Yes, and? This PR is not about bugfixes or any new features. It has only administrative character so that we can tag develop. If there are any issues that need to be solved, then they will happen in other PRs.

Reply:
The change was made in `salsah1` and not in `webapi`. I changed the version to the same version we use in `webapi`. Akka is used in `salsah1` as a simple web server.

The stress tests for knora are on my todo list. I didn't forget about them.

Reply:
Ahh, it's for salsah, I thought it was for webapi.

> The stress tests for knora are on my todo list. I didn't forget about them.

Ok, tell me if I can help with a sample import process